### ISSUE TITLE
Logging prints %2e in constructor path

### ISSUE THREAD

**Issue by breerly** [uber-go/fx#538]
Looks like Fx's output contains `%2egit` instead of `.git`:
  
```
2017/06/29 09:01:39 [Fx] PROVIDE        fx.Lifecycle <= code.uber.internal/devexp/learnfx/vendor/go.uber.org/fx.New.func1()
2017/06/29 09:01:39 [Fx] PROVIDE        envfx.Result <= code.uber.internal/devexp/learnfx/vendor/code.uber.internal/go/envfx%2egit.module()
2017/06/29 09:01:39 [Fx] PROVIDE        configfx.Result <= code.uber.internal/devexp/learnfx/vendor/code.uber.internal/go/configfx%2egit.Module()
```

This is likely due to some escaping.

**Comment by glibsm** [comment]
removing this and a couple other issues from rc2, since they are not affecting the APIs at all and can be done at any point in the release cycle.

**Comment by breerly** [comment]
This can actually be fixed *post* 1.0 since there are no user-facing knobs here.
